docs/defaults-detailed.rst

```diff
@@ -183,7 +195,7 @@ General interface parameters
   - ``auto``: bring the interface up at boot time by the ``networking``
```

Comment:
Is there a default value for "allow"? I think it would be good to put the default value here into the docs.